File: src/main/kotlin/exam2/Exam2.java
package exam2;

public class Exam2 {
	public static void main(String[] args) {
		int number = 15;

		if (number % 2 == 0) {
			System.out.println("Even Number");
		} else {
			System.out.println("Odd Number");
		}

		String str = number % 2 == 0 ? "even" : "odd"; // 3항 연산자로도 표현 가능
	}
}

File: src/main/kotlin/exam2/Exam2.kt
fun main() {
    val number = 15

    val str = if (number % 2 == 0) "even" else "odd"

    if (number % 2 == 0) {
        print("2의 배수")
    } else if (number % 3 == 1) {
        print("3의 배수")
    } else if (number % 5 == 1) {
        print("5의 배수")
    }

    when {
        number % 2 == 0 -> print("2의 배수")
        number % 3 == 0 -> print("3의 배수")
        number % 5 == 0 -> print("5의 배수")
    }

    when (number) {
        1 -> print("1")
        2 -> print("2")
        3 -> print("3")
    }
}
